doc/python/px-arguments.md

```diff
@@ -101,33 +115,55 @@ fig.show()
 Many more examples of wide-form and messy data input can be found in our [detailed wide-form support documentation](/python/wide-form/).
 
 
-### Input Data as Pandas `DataFrame`s
+## DataFrame Input
+
+The first argument of every `px` function is `data_frame`. If you provide a DataFrame as a `px` function's first argument, you can then specify column names as strings from the DataFrame as other arguments.
+
+### Supported DataFrames
+
+`px` functions natively support the following types of DataFrames:
+
+- pandas
```

Comment:
"Pandas" rather than "pandas" (?) not sure what the capitalization should be

Reply:
I say "pandas" as that's how it is in their docs: https://pandas.pydata.org/docs/user_guide/index.html